crm_claim_analytic/tests/test_crm_claim_analytic.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019 Oihane Crucelaegui - AvanzOSC
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests import common
+
+
+class TestCrmClaimAnalytic(common.SavepointCase):
```

Comment:
Class SavepointCase is deprecated, please change to TransactionCase.

And later rebase.